src/components/molecules/UiPhoneNumber/UiPhoneNumber.vue

```diff
@@ -105,6 +103,13 @@ const modelValue = computed({
   get: () => props.modelValue,
   set: (value) => emit('update:modelValue', value),
 });
+const attrs = useAttrs() as { class:string };
+const hasError = computed(() => {
+  if (!attrs.class) {
+    return false;
+  }
+  return !!attrs.class.match(/ui-phone-number--has-error/gm);
+});
```

Comment:
It can be simplified
```suggestion
const hasError = computed(() => (!attrs.class) 
? false : !!attrs.class.match(/ui-phone-number--has-error/gm));
```